### PR TITLE
Single docker to rule all crawlers

### PR DESCRIPTION
**Pull request by mariopino**
Refactor to use one single node process to run all crawlers. And a bunch of other fixes and refactors.

Now you can just simply run `node index.js` to see all that asynchronous magic! ^^

Pending:

- [x] Simplify docker-compose

### CODE REVIEW

File: backend.config.js [hunk 1]
```diff
@@ -1,6 +1,4 @@
 module.exports = {
-  // Enable CORS
-  enableCORS: true,
```
Comment (Bitcoinera) [inline backend.config.js]: Why are you removing CORS here? Is it necessary due to the other changes?

File: index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+const config = require('./backend.config.js');
+const BackendV3 = require('./lib/BackendV3.js');
+
+async function main () {
+  const backendV3 = new BackendV3(config); 
+}
```
Comment (Bitcoinera) [inline index.js]: Ah OK, I see. Forget my comment above

File: lib/crawlers/rewards.js [hunk 3]
```diff
@@ -51,7 +51,9 @@ module.exports = {
       // Fetch last reward events from PolkaScan
       const response = await axios.default.get(`https://api-01.polkascan.io/kusama/api/v1/event?&filter[module_id]=staking&filter[event_id]=Reward`);
 
-      const rewardEvents = response.data.data;
+      const rewardEvents = response.data;
```
Comment (Bitcoinera) [inline lib/crawlers/rewards.js]: This is actually wrong. I just added a commit to fix it

Comment (mariopino) [reply lib/crawlers/rewards.js]: I'm testing why rewards are not inserted in db ;-)

Comment (Bitcoinera) [reply lib/crawlers/rewards.js]: For me it is successfully storing in the local DB:

image

**Pull request merged**